assets/styles/main.css

```diff
@@ -0,0 +1,171 @@
+body {
+  overflow-x: hidden;
+}
+
+main {
+  text-align: left;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.flex-h,
+.flex-v {
+  display: flex;
+}
+
+.flex-v {
+  flex-direction: column;
+}
+
+.flex-h {
+  display: flex;
+  flex-direction: row;
```

Comment:
Flex boxes have a row direction by default
```suggestion
```

Reply:
Thank you for your attention, I fix all issues